src/main/java/com/cottonfactory/products/repositories/JacketRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.cottonfactory.products.repositories;
+
+import com.cottonfactory.products.entities.Jacket;
+import org.springframework.data.jpa.repository.JpaRepository;
+
```

Comment:
Repository annotation is missing

Reply:
it should get invoked from extension of JPA repository